### ISSUE TITLE
Why is onNavigationViewControllerCreated reqired?

### ISSUE THREAD

**Issue by ziga-hvalec** [googlemaps/react-native-navigation-sdk#356]
We just want to use a map to show some markers. We're not using navigation on that screen. 

When doing something like, typescript complains.

```js
<NavigationView
          mapViewCallbacks={mapViewCallbacks}
          onMapViewControllerCreated={setMapViewController}
          // onNavigationViewControllerCreated={() => {}}
/>
```

For now we have an empty controller 🤷

**Comment by jokerttu** [comment]
Yes, this should not be mandatory; marking this as feature-request
You could also use MapView instead of NavigationView if navigation should not be visible on the view:
https://github.com/googlemaps/react-native-navigation-sdk?tab=readme-ov-file#add-a-map-view

**Comment by ziga-hvalec** [comment]
I was not aware this was now available.
I was told previously to just use navigation View without navigation :D
I'll try Map View instad. In this case you can ignore this. since Navigation view should include this parameter.
You can close this now
